src/helpers/firmware/installOsuFirmware.js

```diff
@@ -43,7 +42,6 @@ export default async (
     await remapError(createDeviceSocket(transport, url).toPromise())
     return { success: true }
   } catch (error) {
-    const result = { success: false, error }
-    throw result
+    throw error
```

Comment:
surounding try catch can be removed too? note that the logic have changed too (no longer throwing the object, which was a bad idea anyway